Plugins/Administration/Administration.cpp

```diff
@@ -264,6 +265,12 @@ Events::ArgumentStack Administration::SetServerName(Events::ArgumentStack&& args
     return Events::Arguments();
 }
 
+Events::ArgumentStack Administration::GetServerName(Events::ArgumentStack&& args)
+{
+    const CExoString serverName = Globals::AppManager()->m_pServerExoApp->GetNetLayer()->GetSessionName();
+    return Events::Arguments(std::string(serverName.m_sString ? serverName.m_sString : ""));
```

Comment:
Just `serverName.CStr()`. Probably don't even need the `std::string` part.

Reply:
using CStr now